src/Tenancy.php

```diff
@@ -135,19 +136,24 @@ public function central(callable $callback)
      * @param callable $callback
      * @return void
      */
-    public function runForMultiple($tenants, callable $callback)
+    public function runForMultiple($tenants, callable $callback, bool $withPending = null)
     {
+        $query = $this->model()->newQuery();
+
+        if (is_bool($withPending) && $this->model()::hasGlobalScope(PendingScope::class)){
+            $query->withPending($withPending);
+        }
         // Convert null to all tenants
-        $tenants = is_null($tenants) ? $this->model()->cursor() : $tenants;
+        $tenants = is_null($tenants) ? $query->cursor() : $tenants;
 
         // Convert incrementing int ids to strings
         $tenants = is_int($tenants) ? (string) $tenants : $tenants;
 
         // Wrap string in array
         $tenants = is_string($tenants) ? [$tenants] : $tenants;
 
-        // Use all tenants if $tenants is falsey
-        $tenants = $tenants ?: $this->model()->cursor();
+        // Use all tenants if $tenants is false
```

Comment:
```suggestion
        // Use all tenants if $tenants is falsy
```

"falsey" was correct

Reply:
`falsy`, not `falsey`

Reply:
okay, changing the suggestion to `falsy`